tests/contrib/sensors/test_file_sensor.py

```diff
@@ -74,10 +75,10 @@ def test_simple(self):
                      ignore_ti_state=True)
 
     def test_file_in_nonexistent_dir(self):
-        dir = tempfile.mkdtemp()
+        dir_ = tempfile.mkdtemp()
```

Comment:
```suggestion
        dir = tempfile.mkdtemp()
```

The diff is much easier to read and easier to review if you don't make changes like this. Please revert it.